<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassManagerDlg</class>
 <widget class="QDialog" name="ClassManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Tournament Classes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_class">
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_class">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_remove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Weight Classes</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QListWidget" name="listWidget_weights">
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_add_weight">
            <property name="toolTip">
             <string>Add a new weight class</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_remove_weight">
            <property name="toolTip">
             <string>Remove the currently selected weight class</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_up">
            <property name="toolTip">
             <string>Add a new weight class</string>
            </property>
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_down">
            <property name="toolTip">
             <string>Add a new weight class</string>
            </property>
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Round Time</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_roundTime">
          <property name="text">
           <string>Round</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="timeEdit_round">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="displayFormat">
           <string notr="true">mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_goldenScore">
          <property name="text">
           <string>Golden Score</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTimeEdit" name="timeEdit_goldenScore">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="displayFormat">
           <string notr="true">mm:ss</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClassManagerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClassManagerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
